tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -130,11 +132,182 @@ void test_minkowski() {
+          grad_unit_normal_one_form, unit_normal_one_form, unit_normal_vector),
+      inverse_spatial_metric);
+
+  CHECK_ITERABLE_APPROX(get(ricci_scalar), DataVector(get(ricci_scalar).size(),
+                                                      0.5 / square(mass)));
+}
+
+void test_minkowski() {
+  // Make surface of radius 2.
+  const auto box =
+      db::create<db::AddTags<StrahlkorperTags::items_tags<Frame::Inertial>>,
+                 db::AddComputeItemsTags<
+                     StrahlkorperTags::compute_items_tags<Frame::Inertial>>>(
+          Strahlkorper<Frame::Inertial>(8, 8, 2.0, {{0.0, 0.0, 0.0}}));
+
+  const double t = 0.0;
+  const auto& cart_coords =
+      db::get<StrahlkorperTags::CartesianCoords<Frame::Inertial>>(box);
+
+  EinsteinSolutions::Minkowski<3> solution{};
+
+  const auto deriv_spatial_metric =
+      solution.deriv_spatial_metric(cart_coords, t);
+  const auto inverse_spatial_metric =
+      solution.inverse_spatial_metric(cart_coords, t);
+
+  const DataVector one_over_one_form_magnitude =
+      1.0 /
+      magnitude(db::get<StrahlkorperTags::NormalOneForm<Frame::Inertial>>(box),
+                inverse_spatial_metric);
+  const auto unit_normal_one_form = StrahlkorperGr::unit_normal_one_form(
+      db::get<StrahlkorperTags::NormalOneForm<Frame::Inertial>>(box),
+      one_over_one_form_magnitude);
+  const auto grad_unit_normal_one_form =
+      StrahlkorperGr::grad_unit_normal_one_form(
+          db::get<StrahlkorperTags::Rhat<Frame::Inertial>>(box),
+          db::get<StrahlkorperTags::Radius<Frame::Inertial>>(box),
+          unit_normal_one_form,
+          db::get<StrahlkorperTags::D2xRadius<Frame::Inertial>>(box),
+          one_over_one_form_magnitude,
+          raise_or_lower_first_index(
+              gr::christoffel_first_kind(deriv_spatial_metric),
+              inverse_spatial_metric));
+
+  const auto unit_normal_vector =
+      raise_or_lower_index(unit_normal_one_form, inverse_spatial_metric);
+  const auto extrinsic_curvature = StrahlkorperGr::extrinsic_curvature(
+      grad_unit_normal_one_form, unit_normal_one_form, unit_normal_vector);
+
+  const auto ricci_scalar = StrahlkorperGr::ricci_scalar(
+      make_with_value<tnsr::ii<DataVector, 3, Frame::Inertial>>(
+          inverse_spatial_metric, 0.0),
+      unit_normal_vector, extrinsic_curvature, inverse_spatial_metric);
+
+  CHECK_ITERABLE_APPROX(get(ricci_scalar),
+                        DataVector(get(ricci_scalar).size(), 0.5));
+}
+}  // namespace TestRicciScalar
 }  // namespace
 
-SPECTRE_TEST_CASE("Unit.ApparentHorizons.ConstantExpansion",
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.Expansion",
+                  "[ApparentHorizons][Unit]") {
+  TestExpansion::test_schwarzschild();
+  TestExpansion::test_minkowski();
+}
+
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.ExtrinsicCurvature",
+                  "[ApparentHorizons][Unit]") {
+  TestExtrinsicCurvature::test_minkowski();
```

Comment:
Test the extrinsic curvature in schwarzschild too

Reply:
Unnecessary. All terms in Minkowski are nontrivial, so Minkowski fully tests the function.